src/OpenDebugAD7/Telemetry/DebuggerTelemetry.cs

```diff
@@ -55,6 +56,7 @@ internal sealed class DebuggerTelemetry
         public const string TelemetrySourceFileMappings = "SourceFileMappings";
         public const string TelemetryMIMode = "MIMode";
         public const string TelemetryStackFrameId = TelemetryExecuteInConsole + ".ExecuteInConsole";
```

Comment:
not your change, but why is TelemetryStackFrameId = "Evaluate.ExecuteInConsole.ExecuteInConsole" ?

Reply:
Should be maybe ".StackFrameId"